addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4837,14 +4840,19 @@ msgctxt "#10007"
 msgid "System information"
 msgstr ""
 
+#: xbmc/guilib/WindowIDs.h
+msgctxt "#10008"	
```

Comment:
Likewise renumbering existing strings to insert a new string is to be avoided, add a new string value at the end of the file. There are some id ranges so far unused to allow for related string to be together and additions, but reuse is avoided. With once offs like these simply add at the end.